cmd/waitcmd.go

```diff
@@ -22,9 +22,7 @@ const (
 )
 
 var (
-	sgxLibNames = []string{"libp11SgxEnclave.signed.so", "libp11sgx.so",
-		"libsgx_dcap_ql.so.1", "libsgx_enclave_common.so.1",
-		"libsgx_pce_logic.so.1", "libsgx_qe3_logic.so", "libsgx_urts.so"}
+	sgxLibNames = []string{"libp11SgxEnclave.signed.so", "libp11sgx.so"}
```

Comment:
why need to remove other libs? It should be used below, I suggest that all libs should be checked before the pods get ready.

Reply:
Because other lib's name may change after SGX SDK updating. Now `libsgx_urts.so` has been change to `libsgx_urts.so.2`.

Reply:
Only the name of `libp11SgxEnclave.signed.so`, `libp11sgx.so` can keep unchanged.